widget/table.go

```diff
@@ -67,6 +90,55 @@ func (t *Table) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
+// Selection will mark the specified cell (at row, col) to be marked as selected.
+func (t *Table) Selection() (row, col int) {
+	return t.selectedRow, t.selectedColumn
+}
+
+// SetSelection will mark the specified cell (at row, col) to be marked as selected.
+func (t *Table) SetSelection(row, col int) {
+	t.selectedRow = row
+	t.selectedColumn = col
+
+	t.scrollToVisible(row, col)
+	if t.moveCallback != nil {
+		t.moveCallback()
+	}
+
+	if t.OnSelectionChanged != nil {
+		t.OnSelectionChanged(row, col)
+	}
+}
+
+func (t *Table) scrollToVisible(row, col int) {
```

Comment:
nit: is there a better name than scrollToVisible? It implies that it will scroll to something that is visible, but only objects in the viewport are visible. scrollToShow row col, or scrollToSelection
```suggestion
func (t *Table) scrollToSelection(row, col int) {
```

Reply:
ScrollToSelection would be more wrong if we take the row, col parameters.
This was intended to be a utility method to scroll the specified index to be visible.
It could be called from a hypothetical "ScrollToTopLeft" whereby it would make (0, 0) visible.

Reply:
Then how about just `scrollTo`?